group_vars/all.yml

```diff
@@ -64,6 +64,7 @@ common_packages:
     - apt-transport-https
     - tar
     - bzip2
+    - libgl1-mesa-glx 
```

Comment:
This is ok here as it's pretty innocuous. But with some of the others we need to be fixing conda upstream.